content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,199 @@
+Gateway Client Certificate Validation - Arko Dasgupta (Tetrate),
+GAMMA - John Howard (Solo.io),
+Conformance Profiles and Reports - Mattia Lavacca (Kong), Christine Kim (Isovalent),
+Session Persistence and BackendLBPolicy - Grant Spence (Red Hat), Gina Yeh (Google),
+TLS Terminology Clarifications - Candace Holman (Red Hat),
+and other reviewer and release note contributors.
+
+In the wake of the GA release of Gateway API last October, the Kubernetes
+SIG Network buoyantly announces the v1.1 release
+of [Gateway API](https://gateway-api.sigs.k8s.io/). Gateway API features for
+**Gateway API for Service Mesh (GAMMA)**, **GRPCRoute**, **ParentRef Port**, 
+and **Conformance Profiles and Reports** will graduate to Standard.
+In addition, **Gateway Client Certificate Verification** and **Session
+Persistence** are adopted as new, Experimental features. Clarifications to TLS
+terminology and gateway merging are added to strengthen the release.
+
+## What's new
+
+### Graduation to Standard
+This release includes the graduation to Standard of four eagerly awaited features.
+This means they are no longer experimental concepts. **Standard** denotes a high
+level of confidence in the API surface and provides guarantees of backward
+compatibility.  Though these features are now considered stable, that does not
+mean that they are complete. These and other features will continue to evolve via
+the Experimental channel. For more information on how all of this works, refer
+to the [Gateway API Versioning Policy](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+
+#### [GAMMA](https://gateway-api.sigs.k8s.io/mesh/)
+Service mesh support in Gateway API allows service mesh users to use the same
+API to manage ingress traffic and mesh traffic, reusing the same policy and
+routing interfaces. In Gateway API v1.1, routes (such as HTTPRoute) can now have
+a `Service` as a `parentRef`, to control how traffic to specific services behave.
+For more information, read the 
+[service mesh](https://gateway-api.sigs.k8s.io/mesh/) documentation or see the
+list of 
+[implementations](https://gateway-api.sigs.k8s.io/implementations/#service-mesh-implementation-status).
+As an example, one could do a canary deployment of a workload deep in an application's call graph with an HTTPRoute as follows:
+
+~~~
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: color-canary
+  namespace: faces
+spec:
+  parentRefs:
+    - name: color
+      kind: Service
+      group: core
+      port: 80
+  rules:
+  - backendRefs:
+    - name: color
+      port: 80
+      weight: 50
+    - name: color2
+      port: 80
+      weight: 50
+~~~
+      
+This would split traffic sent to the color Service in the faces namespace 50/50 between the original color Service and the color2 Service, using a portable configuration that's easy to move from one mesh to another.
+Leading Contributors: Flynn, John Howard, Keith Mattix, and Mike Morris
```

Comment:
```suggestion
This would split traffic sent to the `color` Service in the `faces` namespace 50/50 between the original `color` Service and the `color2` Service, using a portable configuration that's easy to move from one mesh to another.

Leading Contributors: Flynn, John Howard, Keith Mattix, and Mike Morris
```

(This adds a blank line parallel to how leading contributors are listed elsewhere, but also sets the names of specific Services and namespaces in code font because, uh, that's how I typically do it at least? 🙂)